bftengine/src/preprocessor/PreProcessor.hpp

```diff
@@ -342,7 +342,9 @@ class PreProcessor {
   // engine supported size is kMaxSubpoolBuffersize_ - responseInternalOverhead_
   static constexpr uint32_t kMaxSubpoolBuffersize_{1UL << 25};
   // Must be sorted in an ascending order, by bufferSize
-  const std::vector<concordUtil::MultiSizeBufferPool::SubpoolConfig> subpoolsConfig_;
+  const concordUtil::MultiSizeBufferPool::SubpoolsConfig dynamicSubpoolsConfig_;
```

Comment:
Why are you keeping it as a member? you are using it only in ctor

Reply:
Removed